storage/objects/generate_v4_get_object_signed_url.go

```diff
@@ -51,7 +51,7 @@ func generateV4GetObjectSignedURL(w io.Writer, bucket, object string) (string, e
 		Expires: time.Now().Add(15 * time.Minute),
 	}
 
-	u, err := client.Bucket(bucket).SignedURL(object, opts)
+	u, err := storage.SignedURL(bucket, object, opts)
```

Comment:
We specifically updated this sample to use `BucketHandle.SignedURL` because it is much more user-friendly for auth, so we think it should be the preferred approach for new users. See #2318 

There is a short example for storage.SignedURL here: https://pkg.go.dev/cloud.google.com/go/storage#hdr-Signed_URLs

Reply:
Hey, sorry just found that my local version of the storage library was an old one. Yea makes sense for me too. Closing PR.

Reply:
No worries! Hope the new version works better for you.